Nio/NewConversation/NewConversationView.swift

```diff
@@ -3,21 +3,49 @@ import MatrixSDK
 
 import NioKit
 
+enum UserStatus {
+    case unkown         // Validity of user is still unkown.
```

Comment:
minor nitpick: small typo

```suggestion
    case unknown         // Validity of user is still unknown.
```